src/sentry/incidents/models.py

```diff
@@ -234,7 +234,15 @@ def snuba_values(self):
         and 'count' keys.
         :return:
         """
-        return {"data": [{"time": time, "count": count} for time, count in self.values]}
+        # We store the values here as floats so that we can support percentage stats.
+        # We don't want to return the time as a float, and to keep things consistent
+        # with what Snuba returns we cast floats to ints when they're whole numbers.
+        return {
+            "data": [
+                {"time": int(time), "count": count if not count.is_integer() else int(count)}
```

Comment:
Why do we need to cast these now?

Reply:
Because the `FloatField` casts everything as a float, so I wanted to try keep things consistent with the way they were before